demos/vc_issuer/src/main.rs

```diff
@@ -67,6 +68,7 @@ thread_local! {
     static SIGNATURES : RefCell<SignatureMap> = RefCell::new(SignatureMap::default());
     static EMPLOYEES : RefCell<HashSet<Principal>> = RefCell::new(HashSet::new());
     static GRADUATES : RefCell<HashSet<Principal>> = RefCell::new(HashSet::new());
+    static ADULTS : RefCell<HashSet<Principal>> = RefCell::new(HashSet::new());
```

Comment:
What about making this a bit more dynamic and adding the age specifically.
So that then the credential would be `VerifiedAge` and the parameter would just be checked against the stored age. I think that would be quite neat. 😉